nextest-runner/src/reporter/displayer/progress.rs

```diff
@@ -521,13 +238,17 @@ impl ProgressBarState {
                 .expect("template is known to be valid"),
         );
 
-        let test_bars =
-            show_running.then(|| TestProgressBars::new(spinner_chars, max_progress_running));
+        let running_tests = show_running.then(Vec::new);
 
         Self {
-            multi_progress,
             bar,
-            test_bars,
+            stats: RunStats::default(),
+            styles: Styles::default(),
```

Comment:
This isn't a huge deal but can you just pass the styles down everywhere as used to be done? Not quite following why that doesn't work any more.

Reply:
The tick event is not going down the same code path as the other events, so I'm not sure at which point it's better to pass these.
Can I push the switch event up to `update_progress_bar` and remove the `tick()` methods?

Reply:
Oh interesting, I see the issue. Honestly feel free to do anything reasonable here, and I'll poke at it later to see if I like something else more. The current implementation is fine.

Reply:
Ok, I'll resist the temptation to try to make it more to my liking then :)